Kwf/Controller/Action/Cli/Web/MaintenanceJobsController.php

```diff
@@ -96,6 +102,28 @@ public function runAction()
                 $nextDailyRun = null;
                 Kwf_Util_Maintenance_Dispatcher::executeJobs(Kwf_Util_Maintenance_Job_Abstract::FREQUENCY_DAILY, $debug, $output);
             }
+
+            Kwf_Component_Data_Root::getInstance()->freeMemory();
+
+            foreach (Kwf_Util_Maintenance_Dispatcher::getAllMaintenanceJobs() as $job) {
+                if ($job->getFrequency() == Kwf_Util_Maintenance_Job_Abstract::FREQUENCY_CUSTOM) {
+                    $jobClass = get_class($job);
+
+                    if (!array_key_exists($jobClass, $lastCustomRun)) $lastCustomRun[$jobClass] = time();
```

Comment:
don't set to current time if job never has been run. I would use null.

getNextRuntime would return something wrong for the very first run.